english/eat-farts/eat-farts

```diff
@@ -0,0 +1,8 @@
+"Pfffffffffffttttttt, pffffffftttttttt." - mr. skeltal
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.